gradle-android-command-plugin/src/test/groovy/com/novoda/gradle/command/AndroidCommandPluginExtensionTest.groovy

```diff
@@ -2,6 +2,7 @@ package com.novoda.gradle.command
 import org.gradle.testfixtures.ProjectBuilder
 
 class AndroidCommandPluginExtensionTest extends GroovyTestCase {
+    static private final String SDK_PATH = '/some/path'
```

Comment:
This should be ordered `private static`.  
See http://docs.oracle.com/javase/7/docs/api/java/lang/reflect/Modifier.html#toString(int)